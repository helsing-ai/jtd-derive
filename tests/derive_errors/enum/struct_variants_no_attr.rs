#[derive(jtd_derive::JsonTypedef)]
enum Foo {
    Bar { x: u32 },
}

fn main() {}
