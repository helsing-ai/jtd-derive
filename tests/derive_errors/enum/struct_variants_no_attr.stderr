error: jtd-derive requires an enum with struct variants to have a tag
 --> tests/derive_errors/enum/struct_variants_no_attr.rs:2:6
  |
2 | enum Foo {
  |      ^^^
